src/main/java/com/superdevs/HealthOMeter/calculator/WHRCalculator.java

```diff
@@ -9,9 +9,9 @@
 import java.math.RoundingMode;
 
 @Component
-public class WhrCalculator {
+public class WHRCalculator {
 
-    Logger logger = LoggerFactory.getLogger(WhrCalculator.class);
+    Logger logger = LoggerFactory.getLogger(WHRCalculator.class);
 
     public BigDecimal calculateWhr (BigDecimal waistRatio, BigDecimal hipRatio){
```

Comment:
calculateWhr > calculateWHR
(camleCase > WaistHipsRatio, imo when named calculateWhr it's not camelCase correct way of naming classes, methods etc.)